lib/order.rb

```diff
@@ -0,0 +1,34 @@
+require 'confirmation_text_sender'
+
+class Order
+
+  attr_reader :order, :total
+
+  def initialize(order, text_confirm_order = ConfirmationTextSender.new)
```

Comment:
I'm confused - what is the order?  I seem to have to initialize and instance of `Order` by passing it an `order` parameter.